src/libraries/Microsoft.Extensions.DependencyModel/src/DependencyContextLoader.cs

```diff
@@ -141,28 +141,30 @@ private string GetDepsJsonPath(Assembly assembly)
 
             string depsJsonFile = Path.ChangeExtension(assemblyLocation, DepsJsonExtension);
             bool depsJsonFileExists = _fileSystem.File.Exists(depsJsonFile);
-
+#if !NET5_0_OR_GREATER
```

Comment:
Why are we making this change?

Reply:
`Assembly.CodeBase` is deprecated in .net5+: https://github.com/dotnet/runtime/issues/31127

Reply:
I don't think removing this code is a good idea for now. You should just suppress the Obsolete warning instead. Ex:

```C#
#pragma warning disable SYSLIB0012 // CodeBase is obsolete
```